src/effects/presets/effectchainpresetmanager.cpp

```diff
@@ -784,6 +789,36 @@ EffectsXmlData EffectChainPresetManager::readEffectsXml(
     return EffectsXmlData{quickEffectPresets, standardEffectChainPresets};
 }
 
+EffectChainPresetPointer EffectChainPresetManager::readEffectsXmlSingleDeck(
+        const QDomDocument& doc, const QString& deckString) {
+    QDomElement root = doc.documentElement();
+
+    auto pQuickEffectChainPreset = getDefaultQuickEffectPreset();
+
+    // Read name of last loaded QuickEffect preset
+    QDomElement quickEffectPresetsElement =
+            XmlParse::selectElement(root, EffectXml::kQuickEffectChainPresets);
+    QDomNodeList quickEffectNodeList =
+            quickEffectPresetsElement.elementsByTagName(
+                    EffectXml::kChainPresetName);
+    for (int i = 0; i < quickEffectNodeList.count(); ++i) {
+        QDomElement presetNameElement = quickEffectNodeList.at(i).toElement();
```

Comment:
This can become a range based loop, to get rid of i.

Reply:
There's QList<type>::iterator, but none for QDomeNodeList.